omf-python/src/geometry.rs

```diff
@@ -72,3 +73,30 @@ impl PySurface {
         PyTriangleArray(self.0.triangles.clone())
     }
 }
+
+#[gen_stub_pyclass]
+#[pyclass(name = "GridSurface")]
+/// A surface defined by a 2D grid a height on each grid vertex.
+pub struct PyGridSurface(pub GridSurface);
+
+#[gen_stub_pymethods]
+#[pymethods]
+impl PyGridSurface {
+    #[getter]
+    fn orient(&self) -> PyOrient2 {
+        PyOrient2(self.0.orient)
+    }
+
+    #[getter]
+    fn grid(&self, py: Python<'_>) -> PyObject {
+        match self.0.grid {
+            omf::Grid2::Regular { .. } => PyGrid2Regular::from(self.0.grid.clone()).into_py(py),
+            omf::Grid2::Tensor { .. } => PyGrid2Tensor::from(self.0.grid.clone()).into_py(py),
+        }
+    }
+
+    #[getter]
```

Comment:
Maybe since this returns a copy of an array we shouldn't make this a getter. That's the current pattern.